src/c_api/c_api_executor.cc

```diff
@@ -849,6 +816,152 @@ int MXExecutorSimpleBindEx(SymbolHandle symbol_handle,
+                           ExecutorHandle* out) {
+  return mxnet::SimpleBindExMaster(symbol_handle,
+                            dev_type, dev_id,
+                            num_g2c_keys, g2c_keys, g2c_dev_types, g2c_dev_ids,
+                            provided_grad_req_list_len, provided_grad_req_names,
+                            provided_grad_req_types,
+                            num_provided_arg_shapes, provided_arg_shape_names,
+                            provided_arg_shape_data, provided_arg_shape_idx,
+                            num_provided_arg_dtypes, provided_arg_dtype_names, provided_arg_dtypes,
+                            num_provided_arg_stypes, provided_arg_stype_names, provided_arg_stypes,
+                            num_shared_arg_names, shared_arg_name_list,
+                            shared_buffer_len, shared_buffer_name_list,
+                            shared_buffer_handle_list, updated_shared_buffer_name_list,
+                            updated_shared_buffer_handle_list,
+                            num_in_args, in_args, arg_grads,
+                            num_aux_states, aux_states,
+                            shared_exec_handle, out);
+}
+
+
+int MXExecutorSimpleBindEx64(SymbolHandle symbol_handle,
+                           int dev_type,
+                           int dev_id,
+                           const uint32_t num_g2c_keys,
+                           const char** g2c_keys,
+                           const int* g2c_dev_types,
+                           const int* g2c_dev_ids,
+                           const uint32_t provided_grad_req_list_len,
+                           const char** provided_grad_req_names,
+                           const char** provided_grad_req_types,
+                           const uint32_t num_provided_arg_shapes,
+                           const char** provided_arg_shape_names,
+                           const int64_t* provided_arg_shape_data,
+                           const uint32_t* provided_arg_shape_idx,
+                           const uint32_t num_provided_arg_dtypes,
+                           const char** provided_arg_dtype_names,
+                           const int* provided_arg_dtypes,
+                           const uint32_t num_provided_arg_stypes,
+                           const char** provided_arg_stype_names,
+                           const int* provided_arg_stypes,
+                           const uint32_t num_shared_arg_names,
+                           const char** shared_arg_name_list,
+                           int* shared_buffer_len,
+                           const char** shared_buffer_name_list,
+                           NDArrayHandle* shared_buffer_handle_list,
+                           const char*** updated_shared_buffer_name_list,
+                           NDArrayHandle** updated_shared_buffer_handle_list,
+                           uint32_t* num_in_args,
```

Comment:
please add api doc

Reply:
@anirudh2290 shouldn't they be same for both 64 and w/o 64 just like in c_api.h ?

Reply:
yes its still good to add it, in case some params change in future. Also one line about what the difference is would be good.

Reply:
It makes sense to add a line in general about APIs with 64 and w/o 64 appended. But if we do it for all its just duplication of documentation, which will lead to increased no. of lines in file. IMO we should simply add a comment to the top of the file saying APIs with 64 appended to them are for large tensor support. @anirudh2290 do you agree? Or do you still want duplicate documentation to be added to each API signature ?

Reply:
i didn't add APIdoc as if would have been same as MXExecutorSimpleBindEx

Reply:
As mentioned offline, I would prefer documentation: one reason is for frontend developers to know what these APIs do, by looking at them in isolation. Also, for docs generated here: https://mxnet.apache.org/api/cpp/docs/api/include_2mxnet_2c__api_8h.html#acecaa93405fc612183fc8780e6d5ae74

Reply:
I have added TODO for myself for adding api docs whereever int64 specific is used. I will shortly send out PR for the same. (Missed ur comment before triggering the CI hence not committing here again)